packages/app/src/pages/Debug.vue

```diff
@@ -1,13 +1,7 @@
 <template>
-  <div
-    v-if="!query.data.value"
-    data-cy="debug-loader"
-  >
-    Loading
-  </div>
   <DebugContainer
```

Comment:
We should follow the same pattern as `Run.vue` utilizing `TransitionQuickFade` to make the transition more smooth:

```jsx
<TransitionQuickFade>
  <DebugLoading v-if="query.fetching.value || !query.data.value" />
  <DebugContainer
    v-else
    data-cy="debug-container"
    :gql="query.data.value"
  />
</TransitionQuickFade>
```

Reply:
Looks much better now 👍🏻